### PR TITLE
Update flake8 config file regarding char limits

### PR DESCRIPTION
**Pull request by gdevincenzi**

### CODE REVIEW

**Comment by rodolfolottin** [comment]
Let's seize this opportunity and set it using `max-line-length=120`. We all know that common sense is not enough.

File: .flake8 [hunk 1]
```diff
@@ -1,4 +1,6 @@
 [flake8]
 ignore =
   # Ignore line limit of 79 characters
+  # Our motivation is that not everyone enjoys this limit. 
+  # Just use common sense and keep it around 100-120
```
Comment (phsetti) [inline .flake8]: You could use something like `max-line-length = 120` here.

**Review by phsetti — APPROVED** [review]

**Comment by gdevincenzi** [comment]
Let me clarify a little bit then:
@rodolfolottin warned me that my linter wasn't checking for line length. I did check that it wasn't and asked @smaniotto why not, and decided to add his reply to the comment in the code.

**Comment by gdevincenzi** [comment]
I think the spirit of it is that you should `hover around` those numbers, but no need to make big adjustments if you went a couple chars above the limit.

Like @smaniotto said to me, we will just end up discussing something that is not that important. No one is writing one liners of 200+ chars, right?

What do you guys think?

**Comment by rodolfolottin** [comment]
We are all complaining about the lack of patterns that we have here at cheesecake; whether frontend, backend and mobile. IMHO defining it is one step ahead in resolving this problem.

**Review by dleitee — APPROVED** [review]

**Comment by fredericojordan** [comment]
I'm with the `max-line-length = 120` bandwagon!

**Review by fredericojordan — CHANGES_REQUESTED** [review]
I'd rather just have a limit set, as people have mentioned before.

**Comment by gdevincenzi** [comment]
I don't mind having this limit set, if that's what we all agree on.

**Comment by gdevincenzi** [comment]
@fredericojordan @rodolfolottin @phsetti @dleitee @smaniotto @douglasgimli @urataki @karranb 

Since it seems like there can be a consensus around the 120 limit, I've changed this commit to reflect that.

If no one disagrees, let's push this change forward.

**Review by fredericojordan — APPROVED** [review]

**Comment by fredericojordan** [comment]
:shipit:

**Review by dleitee — APPROVED** [review]

**Comment by rodolfolottin** [comment]
:shipit:

**Review by rodolfolottin — APPROVED** [review]

**Pull request merged**